crates/parcel_monitoring/src/tracer.rs

```diff
@@ -1,16 +1,15 @@
 use std::sync::Arc;
 
-use napi::Env;
-use napi::JsObject;
+use anyhow::anyhow;
 use serde::Deserialize;
 use serde::Serialize;
 use tracing_appender::non_blocking::WorkerGuard;
 
+use crate::from_env::{optional_var, FromEnvError};
+
 #[derive(Debug, Deserialize, Serialize)]
 #[serde(rename_all = "camelCase", tag = "mode")]
-pub enum TracerMode {
-  /// Disable the Tracer
-  Disabled,
+pub enum TracingOptions {
```

Comment:
How come you favour `Options` over `Mode`? The latter makes more sense to me, since there's only one variant to be used at a time

Reply:
Changed back